### DISCUSSION TITLE
how can i stop one of tickers?

### DISCUSSION THREAD

**Discussion by yangliguo7** [pixijs/pixijs#7987] [Q&A] [answered]
In my project，
I have some tickers sometimes。like
`const tickerX =  app.ticker.add(()=>{
    fn1()
});
const tickerY  = app.ticker.add(()=>{
    fn2()
});...`
But，How can I stop the `tickerY `?
I try to use `tickerY.stop()  `，but all of Tickers is stop。。。。
so.
how can i stop one of tickers?
or
Is it the only way to start a task by using app.tickers

**Comment by bigtimebuddy** [comment, marked as answer]
Please refer to the documentation for Ticker: http://pixijs.download/dev/docs/PIXI.Ticker.html#add

The `add()` method returns the instance of the Ticker. If you want to remove a single ticker you should do this:

```js
// Add the ticker
app.ticker.add(fn1);

// Remove the ticker
app.ticker.remove(fn1);
```

**Reply by yangliguo7** [reply]
This really helped me a lot, thank you so much
